morecantile/models.py

```diff
@@ -149,12 +149,21 @@ def __init__(self, **data):
         """Create PyProj transforms and check if TileMatrixSet supports quadkeys."""
         super().__init__(**data)
         self._is_quadtree = check_quadkey_support(self.tileMatrix)
-        self._to_wgs84 = Transformer.from_crs(
-            self.supportedCRS, WGS84_CRS, always_xy=True
-        )
-        self._from_wgs84 = Transformer.from_crs(
-            WGS84_CRS, self.supportedCRS, always_xy=True
-        )
+        try:
+            self._to_wgs84 = Transformer.from_crs(
+                self.supportedCRS, WGS84_CRS, always_xy=True
+            )
+            self._from_wgs84 = Transformer.from_crs(
+                WGS84_CRS, self.supportedCRS, always_xy=True
+            )
+        except ProjError:
```

Comment:
Sometime (e.g working with non earth projection) we cannot create Transformer with WGS84 srs